### PR TITLE
fix conflicting tx result messages

### PR DESCRIPTION
**Pull request by Kevingislason**
## Description

closes: #387 

---

Before we can merge this PR, please make sure that all the following items have been checked off. If any of the checklist items are not applicable, please leave them but write a little note why.

- [x] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work
- [x] Re-reviewed files changed in the Github PR explorer
 
## If PR to `staging`
- [x] Updated relevant documentation in the code
- [x] Test code changes manually, and describe your procedure in a comment on this PR
- [x] Mark this PR with the correct milestone

### CODE REVIEW

**Comment by jincan39** [comment]
LGTM

File: src/pages/SendPage/SendContext/index.tsx [hunk 1]
```diff
@@ -401,6 +401,11 @@ export const SendContextProvider = (props) => {
         }
       }
     } else if (status.isFinalized) {
+      for (const event of events) {
+        if (api.events.utility.BatchInterrupted.is(event.event)) {
+          return;
```
Comment (zhangjing28) [inline src/pages/SendPage/SendContext/index.tsx:406]: Why not set `txStatus` to fail in this condition?

Comment (Kevingislason) [reply src/pages/SendPage/SendContext/index.tsx:406]: The body of `handleTxRes` runs the first time when the transaction is put into a block, and a second time when the transaction's block is finalized. If a transaction fails, we will know that the transaction failed once it is in a block. So by the time a failed transaction is finalized, we have already handled the failure and surfaced it to the UI. So this code addition prevents the transaction from being handled twice.

Comment (zhangjing28) [reply src/pages/SendPage/SendContext/index.tsx:406]: Got it

**Review by zhangjing28 — APPROVED** [review]

**Pull request merged**